docs/UserGuide.adoc

```diff
@@ -298,32 +340,38 @@ There is no need to save manually.
 
 == Command Summary
 
-* *Add* `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY(DD/MM/YYYY)] [w/WEBSITE] [t/TAG]...` +
+* *Help* : `help` +
+* *Add* `add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [w/WEBSITE] [t/TAG]...` +
 e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 b/20/10/2000 w/https://www.yahoo.com/ t/friend t/colleague` +
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
-e.g. `delete 3` +
+* *List* : `list` +
+* *Sort* : `sort` +
 * *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [r/REMARK] [w/WEBSITE] [t/TAG]...` +
 e.g. `edit 2 n/James Lee e/jameslee@example.com` +
+* *Profile*: `profile C:\Users\User\Pictures\Saved Pictures\alex.jpg` +
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
 e.g. `find James Jake`
-* *List* : `list` +
-* *Help* : `help` +
+* *Delete* : `delete INDEX` +
+e.g. `delete 3` +
 * *Select* : `select INDEX` +
 e.g.`select 2`
+* *Remark* : `remark index r/Likes to drink coffee` +
 * *History* : `history` +
 * *Undo* : `undo` +
 * *Redo* : `redo` +
+* *Clear* : `clear` +
+* *Theme* : `theme light` +
+* *Exit* : `exit` +
+
 
 == Parameters
-*Name*: n/ +
-*Phone Number*: p/ +
-*Email*: e/ +
-*Address*:  a/ +
-*Birthday*: b/ +
-*Remark*^1^: r/ +
-*Website*: w/ +
-*Tags*^2^: t/ +
+*Name*: n/alphanumeric characters and spaces +
+*Phone Number*: p/at least 3 digits long +
+*Email*: e/ 2 alphanumeric/period strings separated by '@' +
+*Address*:  a/Address of person (can be any value) +
+*Birthday*: b/(DD/MM/YYYY) +
+*Remark*^1^: r/Can be any value or blank +
+*Website*: w/Should be a valid website +
```

Comment:
For now should add "beginning with http:// or https://  cause we don't have the auto append yet